open_bounties/0119 Frontend POAP Web APP.md 

```diff
@@ -0,0 +1,63 @@
+---
+ID: 0079
```

Comment:
Probably for a separate PR, but I've never used this ID field and I think we should probably delete it.